### ISSUE TITLE
ENOENT on first run

### ISSUE THREAD

**Issue by peterennis** [ionic-team/ionic-pwa-toolkit#52]
ENOENT on first run
Ctr-C, then it works on second run
Screen flashes twice on opening

**Stencil version:** (run `npm list @stencil/core` from a terminal/cmd prompt and paste output below):

```
C:\ae\aedev\ionic-pwa-toolkit>npm list @stencil/core
@stencil/starter@0.0.1 C:\ae\aedev\ionic-pwa-toolkit
`-- @stencil/core@0.7.24
```

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
```
{
  "error": {
    "errno": -4058,
    "code": "ENOENT",
    "syscall": "stat",
    "path": "C:\\ae\\aedev\\ionic-pwa-toolkit\\www"
  }
}
```

![enoent](https://user-images.githubusercontent.com/140737/38775776-47d73cc2-403f-11e8-95ca-233382fc01e8.gif)

**Expected behavior:**
Should work on first run
Screen should not flash twice

**Steps to reproduce:**
Create 

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Comment by scottopolis** [comment]
I got this same error, although the errno was a different number.

**Comment by anilbhanushali** [comment]
I am also getting the same error on browser and on terminal as below
<img width="897" alt="screenshot 2018-05-27 18 09 03" src="https://user-images.githubusercontent.com/7549804/40586035-15b7fae0-61d9-11e8-98d6-19a10c8992db.png">

**Comment by adamdbradley** [comment]
This should be fixed with the intergrated dev-server, as of v0.10.0.

@anilbhanushali The `semver` error should be fixed in the next release, thanks.
